apps/prairielearn/src/ee/lib/billing/plan-grants.ts

```diff
@@ -15,29 +17,43 @@ import {
 import { PlanFeatureName, planGrantsMatchPlanFeatures } from './plans-types';
 import { features } from '../../../lib/features';
 
-function userHasRole(res: Response) {
+type ResLocals = Record<string, any>;
+
+function userHasRole(authz_data: any) {
```

Comment:
I have half a mind to define a Zod schema/type for `authz_data`...

Reply:
Oh yes please!! I always need to print it out to find out what's in there 😭